RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEGenericESProducer.cc

```diff
@@ -45,9 +45,9 @@ PixelCPEGenericESProducer::PixelCPEGenericESProducer(const edm::ParameterSet& p)
   // Use LA-width from DB. If both (upper and this) are false LA-width is calcuated from LA-offset
   useLAWidthFromDB_ = p.getParameter<bool>("useLAWidthFromDB");
   // Use Alignment LA-offset
-  const bool useLAAlignmentOffsets = p.getParameter<bool>("useLAAlignmentOffsets");
+  const bool doLorentzFromAlignment = p.getParameter<bool>("doLorentzFromAlignment");
```

Comment:
there is a commented out cout on L70 which is using `useLAAlignmentOffsets_` (already broken, but still close/recognizable); please update there as well or drop that cout

Reply:
Ok I removed the cout